src/views/ipv/page/pyi-continue-with-driving-licence.njk

```diff
@@ -0,0 +1,56 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% set pageTitleName = 'pages.pyiContinueWithDrivingLicence.title' | translate %}
+{% set googleTagManagerPageId = "pyiContinueWithDrivingLicence" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.pyiContinueWithDrivingLicence.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pyiContinueWithDrivingLicence.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#pyiContinueWithDrivingLicenceForm" %}
+
+{% block content %}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pyiContinueWithDrivingLicence.header' | translate }}</h1>
+  <p class="govuk-body">{{ 'pages.pyiContinueWithDrivingLicence.content.paragraph1' | translate | safe }}</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>{{ 'pages.pyiContinueWithDrivingLicence.content.bullet1' | translate | safe }}</li>
+    <li>{{ 'pages.pyiContinueWithDrivingLicence.content.bullet2' | translate | safe }}</li>
+  </ul>
+  <p class="govuk-body">{{ 'pages.pyiContinueWithDrivingLicence.content.paragraph2' | translate | safe }}</p>
+  <p class="govuk-body">{{ 'pages.pyiContinueWithDrivingLicence.content.paragraph3' | translate | safe }}</p>
```

Comment:
Do we need the `safe` nunjucks filter here?

I thought we only needed it to escape special characters in the translation strings (i.e. if we need to add an anchor tag/bold halfway through a string)

Possibly I've misunderstood, but I'm not sure we need it :)

Reply:
We are saying this output is safe so Nunjucks not to escape this output.

Reply:
Ah yes sorry think I didn't explain that well. 

Nunjucks will escape special characters by default to prevent us from writing html instead of strings. Putting `| safe` in is essentially telling Nunjucks that the string contains special characters and we want to render it as html.

But I'm not sure it's necessary here, because our string doesn't contain any special characters that we would need to escape, so we don't need to use the safe filter.

Reply:
@RogVanGDS  seems we may need it for welsh translations e.g â'ch

Reply:
As discussed removed the safe filter from both the pages.